.github/codecov.yml

```diff
@@ -0,0 +1,25 @@
+coverage:
```

Comment:
I haven't seen anyone look at this and am not particularly convinced code coverage is a good metric for determining quality. I would be fine dropping this.

Reply:
Perhaps this would be better to discuss post merge of this tho.

Reply:
Captured this in #9128.